src/strategies/passport/passport-auth0/auth0-strategy-factory-provider.ts

```diff
@@ -0,0 +1,104 @@
+import {Provider, inject} from '@loopback/core';
+import {HttpErrors} from '@loopback/rest';
+import {Strategies} from '../../../keys';
+import {VerifyFunction} from '../../../types';
+import {AuthErrorKeys} from '../../../error-keys';
+import {
+  ExtraVerificationParams,
+  Profile,
+  Strategy,
+  StrategyOption,
+  StrategyOptionWithRequest,
+} from 'passport-auth0';
+import {Request} from 'express';
+
+export interface Auth0StrategyOptions extends StrategyOption {
+  passReqToCallback?: false | undefined;
+}
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export type VerifyCallback = (err?: any, user?: any, info?: any) => void;
```

Comment:
Isnt this available in passport strategy typings

Reply:
no not explicitly